content/guides/embed/ui-elements/explorer.md

```diff
@@ -264,7 +271,175 @@ more, see [Dedicated Scopes for Box UI Elements][scopes].
 | User want basic functionality + preview + download + rename file/folder names             | `base_explorer` + `item_preview` + `item_download` + `item_rename`                                                |
 | User wants all functionality (basic, preview, download, rename, share, upload and delete) | `base_explorer` + `item_preview` + `item_download` + `item_rename` + `item_delete` + `item_share` + `item_upload` |
 
+## Metadata view
+
+With Content Explorer you can also display files
+and folders based on their metadata.
+This view is called the metadata view and uses
+metadata template and metadata query to find the data you
+want to display.
+
+### Prerequisites
+
+- Node version 18.18.2 or higher
+- React version 17.0.2 or higher (React 18 is not supported)
+- BUIE version 19.0.0 or higher
+
+### Create and configure an app
+
+  1. [Create a Box app][box-app].
+  2. Add the local development address in the CORS Domains:
+    ![CORS Domains](./images/box-app-cors.png)
+  3. Generate a [developer token][token].
+
+### Create a metadata template
+
+The next step is to create a metadata template you will use to populate the Content Explorer.
+
+1. Create a metadata template. You can use [Metadata API][creating-templates-api] or [Admin Console][creating-templates-ui] to do so.
```

Comment:
Theoretically they could also do it with CLI, not sure if you want to mention it